client/components/badge/style.scss

```diff
@@ -0,0 +1,22 @@
+.badge {
+	display: inline-block;
+	border-radius: 100px;
```

Comment:
Why is this 100px? It looks randomly picked number. Shouldn't it match the height of the badge?

Reply:
It's meant to look like a pill, so for higher badge we want more rounding - I assumed no badge will get higher than 100px so that's where this number comes from. I could also make it 1000px or so - I don't think we can match the exact height since 100% does something a bit different.

Reply:
Ah, I thought the height of the badge is always the same. My bad, sorry!

Reply:
Sorry late comment but I believe `border-radius: 50%` would've read more clearly here.